src/node_api.h

```diff
@@ -54,11 +55,20 @@ typedef struct napi_module {
 #define NAPI_MODULE_INITIALIZER_BASE napi_register_module_v
 #endif
 
+#define NAPI_MODULE_GET_API_VERSION_BASE napi_module_get_api_version_v
```

Comment:
Since this is a new Node-API-related preprocessor directive in a public header, can we rename it and all newly added preprocessor directives to `NODE_API_*` (`NODE_API_MODULE_GET_API_VERSION_BASE` in this case and `NODE_API_MODULE_GET_API_VERSION` below)?

Reply:
Good point! I will do it.

Reply:
Changed all `napi` prefixes to `node_api` for the new APIs.